airflow/www/views.py

```diff
@@ -702,6 +702,22 @@ def _iter_parsed_moved_data_table_names():
                 # Second segment is a version marker that we don't need to show.
                 yield segments[2], table_name
 
+        warn_deployment_query = session.query(Log).filter(Log.event == "robots").count()
```

Comment:
Should this be limited to the last week or something? The warning message explicitly states that `Recent requests have been made to /robots.txt` but this could be triggered by very old requests. 

There's also no way to clear this entry without deleting the log event so it might persist for a long time, even if the vulnerability is fixed.

Reply:
Wondering if it would be good enough to clear requests older than a week and only display the message on recent requests

Reply:
Could we also just query for events within one week old? Something like:

```python
warn_deployment_query = (
    session.query(Log)
    .filter(Log.event == "robots")
    .filter(Log.dttm > (utcnow() - timedelta(days=7)))
    .count()
)
```